packages/data-point/README.md

```diff
@@ -1510,6 +1510,42 @@ dataPoint.addEntities({
   ```
 </details>
 
+Note: We recommend using reducer as an alias for transform
```

Comment:
The plan according to the PR is to move away from transform and towards using reducer from what I understand. So it makes sense to add a console warning. Thanks 👍

Reply:
I'm not sure the best place for the warning. @acatl or @raingerber?

Reply:
I'm not sure if @acatl wants to deprecate `transform` or not - but if we do, I think it's enough to mention that in the readme (i.e. `this will be deprecated in the future`) because we would be providing a codemod to make the deprecation as easy as possible